src/content/tutorials/observability/multi-tenancy/creating-grafana-organization/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+linkTitle: Creating a Grafana organization
+title: Creating a Grafana organization
+description: Guide explaining how to manage Grafana organizations in the Observability Platform.
+menu:
+  principal:
+    identifier: tutorials-observability-multitenancy-create-grafana-organization
+    parent: tutorials-observability-multitenancy
+weight: 40
+last_review_date: 2024-12-12
+user_questions:
+  - How to create a grafana organization?
+  - How to access multi-tenant observability data?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+When you first access [your installations `Grafana`]({{< relref "/tutorials/observability/data-exploration/accessing-grafana" >}}), you will be accessing an organization named _Shared Org_, which gives you access to a set of curated dashboards. However, if you want to provide multi-tenancy of your observability data (isolation between tenants), the observability platform allows you to create your own organizations in self-service.
```

Comment:
If it's not already written / planned for another resource, it would be good to include a note about what multi-tenancy actually is in this context. e.g.
```suggestion
Grafana Organizations provide logical separation of dashboards, annotations, and other Grafana resources shown in the Grafana UI. When initially [accessing an installation's Grafana instance]({{< relref "/tutorials/observability/data-exploration/accessing-grafana" >}}), the default _Shared_ organization provides access to a set of common, curated dashboards visible to all users. The observability platform allows self-service creation of additional organizations to provide more a more tailored experience for platform users and more granular control for platform administrators.

Note: data from all organizations remains stored locally in the management cluster. Creating a Grafana Organization should be considered a privileged operation, since it allows access to all log and metric data in the MC.
```